[2025-08-22T06:01:58.062+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-08-22T06:01:58.077+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_stg.create_and_load_data_to_stg manual__2025-08-22T06:01:47+00:00 [queued]>
[2025-08-22T06:01:58.085+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_stg.create_and_load_data_to_stg manual__2025-08-22T06:01:47+00:00 [queued]>
[2025-08-22T06:01:58.085+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 1
[2025-08-22T06:01:58.096+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): create_and_load_data_to_stg> on 2025-08-22 06:01:47+00:00
[2025-08-22T06:01:58.107+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2468) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-22T06:01:58.109+0000] {standard_task_runner.py:72} INFO - Started process 2470 to run task
[2025-08-22T06:01:58.110+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_stg', 'create_and_load_data_to_stg', 'manual__2025-08-22T06:01:47+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline_stg.py', '--cfg-path', '/tmp/tmpv9guh7r3']
[2025-08-22T06:01:58.113+0000] {standard_task_runner.py:105} INFO - Job 51: Subtask create_and_load_data_to_stg
[2025-08-22T06:01:58.157+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_pipeline_stg.create_and_load_data_to_stg manual__2025-08-22T06:01:47+00:00 [running]> on host 21c40e4c1785
[2025-08-22T06:01:58.236+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_stg' AIRFLOW_CTX_TASK_ID='create_and_load_data_to_stg' AIRFLOW_CTX_EXECUTION_DATE='2025-08-22T06:01:47+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-22T06:01:47+00:00'
[2025-08-22T06:01:58.237+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-08-22T06:01:58.283+0000] {base.py:84} INFO - Retrieving connection 'datalake_1'
[2025-08-22T06:01:58.387+0000] {logging_mixin.py:190} INFO - Data loaded to STG schema for special_ships in table 'special_ships_stg' successfully.
[2025-08-22T06:01:58.452+0000] {logging_mixin.py:190} INFO - Data loaded to STG schema for merchant_ships in table 'merchant_ships_stg' successfully.
[2025-08-22T06:01:58.511+0000] {logging_mixin.py:190} INFO - Data loaded to STG schema for offshore_plants in table 'offshore_plants_stg' successfully.
[2025-08-22T06:01:58.512+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-08-22T06:01:58.522+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-08-22T06:01:58.522+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_stg, task_id=create_and_load_data_to_stg, run_id=manual__2025-08-22T06:01:47+00:00, execution_date=20250822T060147, start_date=20250822T060158, end_date=20250822T060158
[2025-08-22T06:01:58.565+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2025-08-22T06:01:58.582+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-22T06:01:58.584+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
